### DISCUSSION TITLE
Is it possible to register the extension app in the Whale browser?

### DISCUSSION THREAD

**Discussion by shing100** [Authenticator-Extension/Authenticator#905] [Q&A]
### Describe the issue

Hello
I'm using the extension app very well.
I really want to use the extension app in Naver Whale browser.
Is it possible to register the extension app in the Whale browser?

https://store.whale.naver.com/extensions/popular

### Browser

Other

### Browser Version

null

### Extension Version

6.3.3

**Comment by Sneezry** [comment]
Hi @shing100 

Thanks for your feedback. We do not have a plan to publish Authenticator extension to other marketplaces, currently. But I can install the extension from Chrome Web Store in NAVER Whale browser successfully.

![image](https://user-images.githubusercontent.com/1621293/169481181-1702204d-5845-4e6a-82fb-f118b35425f4.png)

Can you have a try and see if it works on your side?

**Comment by shing100** [comment]
![image](https://user-images.githubusercontent.com/19279129/169695504-d2eff611-544d-4919-9eeb-98753fa82f2c.png)

thanks
It is installed normally and works well.
